templates/search.html

```diff
@@ -11,7 +11,26 @@
         <!-- Your HTML should start here -->
         <!-- Look up the Jinja documentation to display items passed from the server -->
         <!-- There should be no JavaScript used on this page -->
-
+        <div class="container">
+            <h1>Search</h1>
+            <form class="search" action="/search">
+                <input type="text" name="query" id="query" placeholder="search">
+            </form>
+            <div class="result-meta">Showing {{results|length}} results for "{{searchstring}}"</div>
+            {% for i in results%}
+            <div class="result">
+                {% if 'types' in i %}
```

Comment:
You might want to consider differentiating your Pokemon results from your Team results in a different way. What if there was a new category of search result that also had a `types` property? Could you create a `result_type` property that is part of your result objects to make it clearer?